server/fleet/queries.go

```diff
@@ -223,6 +234,22 @@ func verifyLogging(logging string) error {
 	return nil
 }
 
+func verifyQueryPlatforms(platforms string) error {
+	if emptyString(platforms) {
+		return nil
+	}
+	platformsList := strings.Split(platforms, ",")
+	for _, platform := range platformsList {
+		switch strings.TrimSpace(platform) {
```

Comment:
Should we accept `darwin<space>,<space>linux,windows`? IMO we shouldn't, so we shouldn't trim here.

Reply:
@zhumo ^ ?

Reply:
This is not something we need to fix now for sure. Something to consider.

Reply:
I would say let's be consistent with the behavior in the rest of the app. which I think would mean doing the trimming.

cc @rachaelshaw